libs/design/src/molecules/feature/feature-title/feature-title.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: '[daff-feature-title]',
+  template: '<ng-content></ng-content>',
+  host: {'class': 'daff-feature__title'}
```

Comment:
changeDetection: ChangeDetectionStrategy.OnPush

Reply:
Resolved in 688add3